metadata-ingestion/docs/sources/mlflow/mlflow_pre.md

```diff
@@ -2,8 +2,10 @@
+|                                   Source Concept                                    |                                              DataHub Concept                                              | Notes                                                                                                                                                                                                                                                                                            |
+|:-----------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------:|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|  [`Registered Model`](https://mlflow.org/docs/latest/model-registry/#registered-model)  |        [`MlModelGroup`](https://datahubproject.io/docs/generated/metamodel/entities/mlmodelgroup/)        | The name of a Model Group is the same as a Registered Model's name (e.g. my_mlflow_model). Registered Models serve as containers for multiple versions of the same model in MLflow.                                                                                                              |
+|   [`Model Version`](https://mlflow.org/docs/latest/model-registry/#model-version)    |             [`MlModel`](https://datahubproject.io/docs/generated/metamodel/entities/mlmodel/)             | The name of a Model is `{registered_model_name}{model_name_separator}{model_version}` (e.g. my_mlflow_model_1 for Registered Model named my_mlflow_model and Version 1, my_mlflow_model_2, etc.). Each Model Version represents a specific iteration of a model with its own artifacts and metadata. |
+|         [`Experiment`](https://mlflow.org/docs/latest/tracking/#experiments)         |           [`Container`](https://datahubproject.io/docs/generated/metamodel/entities/container/)           | Each Experiment in MLflow is mapped to a Container in DataHub. Experiments organize related runs and serve as logical groupings for model development iterations, allowing tracking of parameters, metrics, and artifacts.                                                                       |
+|            [`Run`](https://mlflow.org/docs/latest/tracking/#runs)             | [`DataProcessInstance`](https://datahubproject.io/docs/generated/metamodel/entities/dataprocessinstance/) | Each Run is mapped to a DataProcessInstance in DataHub, capturing the execution details, parameters, metrics, and lineage to a model.                                                                                                                                                            |
```

Comment:
I'm a little hesitant to add other details here (e.g. how we map specific aspects) since they might change in the future, so I just added high-level entities.